### PR TITLE
(SERVER-2795) Fix CSR deletion and cleanup whitespace

### PR DESCRIPTION
**Pull request by Magisus**
This commit fixes a misplaced parenthesis that made it impossible to
delete CSRs via the API. It also cleans up some erroneous whitespace.

### CODE REVIEW

**Comment by Magisus** [comment]
So we have [tests](https://github.com/puppetlabs/puppetserver/blob/master/test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj#L176) that call the handlers for this function directly, but nothing for the CSR HTTP API. Do we think it's worth adding those?

**Comment by Magisus** [comment]
I had a test that used mock JRubies, and I just found where `autosign=true` was coming from in that context: https://github.com/puppetlabs/puppetserver/blob/master/test/unit/puppetlabs/services/jruby/jruby_puppet_testutils.clj#L232. We do not provide good mechanisms for overriding things in that hash. I think changing that would be the most straightforward solution here.

**Comment by mwaggett** [comment]
I think it would be good to add some sort of test that would've caught this typo, whatever that looks like.

**Comment by Magisus** [comment]
Verified that this test failed without the fix.

File: src/clj/puppetlabs/services/ca/certificate_authority_core.clj [hunk 1]
```diff
@@ -341,9 +341,9 @@
         (GET [""] [subject]
           (handle-get-certificate-request subject ca-settings))
         (PUT [""] [subject :as {body :body}]
-          (handle-put-certificate-request! subject body ca-settings)))
+          (handle-put-certificate-request! subject body ca-settings))
         (DELETE [""] [subject]
-          (handle-delete-certificate-request! subject ca-settings))
+          (handle-delete-certificate-request! subject ca-settings)))
```
Comment (justinstoller) [inline src/clj/puppetlabs/services/ca/certificate_authority_core.clj:346]: 🤦

Comment (Magisus) [reply src/clj/puppetlabs/services/ca/certificate_authority_core.clj:346]: Pretty much.

**Pull request merged**